src/Hazelcast.Net.Tests/Remote/MapTests.cs

```diff
@@ -298,11 +298,11 @@ public async Task SetWithTimeToLive()
             // and does not return anything
             // NOTE: no way to know whether it added or replaced?
 
-            await map.SetAsync("key", 42, TimeSpan.FromSeconds(1)).CfAwait();
+            await map.SetAsync("key", 42, TimeSpan.FromSeconds(2)).CfAwait();
             var value = await map.GetAsync("key").CfAwait();
             Assert.AreEqual(42, value);
 
-            await Task.Delay(1500); // wait for 1.5 second
+            await Task.Delay(3000); // wait for 3 second
```

Comment:
Why change these timeouts?

Reply:
I hate these tests. Testing for TTL takes time and is totally unreliable. What happened was, on GitHub environment, the test could run so slow that the TTL would expire on the server before we have a chance to get the value back. So... I changed them. All those tests that rely on timing are really bad but I don't know how do to it differently?